source/_integrations/zha.markdown

```diff
@@ -138,6 +138,25 @@ enable_quirks:
 
 To add new devices to the network, call the `permit` service on the `zha` domain. Do this by clicking the Service icon in Developer tools and typing `zha.permit` in the **Service** dropdown box. Next, follow the device instructions for adding, scanning or factory reset.
 
+### OTA firmware updates
+
+ZHA component does have the ability to automatically download and perform OTA (Over-The-Air) firmware updates of Zigbee devices if the OTA firmware provider source URL for updates is available. OTA firmware updating is set to disabled (`false`) in the configuration by default.
+
+Currently, OTA providers for firmware updates are only available for IKEA and LEDVANCE devices. OTA updates for device of other manufactures could possible also be supported by ZHA dependencies in the future, if these manufacturers publish their firmware publicly.
+
+To enable OTA firmware updates for the ZHA integration you need to add the following configuration to your `configuration.yaml` and restart Home Assistant:
+
+```yaml
+ota:
```

Comment:
The `ota:` key should be under 
```
zha:
    zigpy_config:
        ota:
```

Reply:
Dang, was on mobile and comment was still pending

Reply:
@Hedda can you create a new PR with this correction?

Reply:
I'll pick that up..

Reply:
PR open: <https://github.com/home-assistant/home-assistant.io/pull/13811>